crates/polars-lazy/src/tests/io.rs

```diff
@@ -225,7 +225,6 @@ fn slice_at_union(lp_arena: &Arena<ALogicalPlan>, lp: Node) -> bool {
 }
 
 #[test]
-#[cfg(not(target_os = "windows"))]
```

Comment:
Why are these removed?

Reply:
As I am working on Windows, and working on the CSV part of the code, I saw that nothing in there prevented the test from running under Windows. (as also evidenced by the successful checks on github actions)

That's why I removed it.

Please do let me know if it is necessary to be there (personal opinion: I doubt it) and I'll add it back.